sport/app/football/controllers/MoreOnMatchController.scala

```diff
@@ -142,6 +143,15 @@ object NxAnswer {
     minByMin.map(x => LinkTo(x.url))
   }
 
+  def makeMatchReportUrl(implicit
```

Comment:
Since you are passing `request`, `theMatch` and `related` explicitly on line 175, you do not need to `implicit` keyword here.  The same is probably true for `makeMinByMinUrl`

Reply:
Actually, this won't work because `request` needs to be `implicit` within this function so that `fetchRelatedMatchContent` gets it

Adam has made a better suggestion to use 2 argument lists here, one for your data (the match and related) and one for your "magic" (the request)

i.e. `def makeMatchReportUrl(theMatch: FootballMatch, related: Seq[ContentType])(implicit request: RequestHeader)`